README.DATUMGRID

```diff
@@ -197,18 +139,33 @@ grid coordinates are referenced to ETRS89.
 
 * fvr09.gtx
 
-*Package*: proj-datumgrid
+## Regional PROJ ressource packages
 
-### Greenland: GRS80 ellipsoidal heights -> GVR2016
+In addition to the core proj-datumgrid package, regional collections of grids
+and init-files also exist. These include grids and init-files that are not
+essential for the functionality of PROJ but still of general interest to
+PROJ-users in the package-specific regions.
 
-*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
-*Format*: GTX  
-*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+The regional packages can be downloaded from the
+[OSGeo download server](http://download.osgeo.org/proj/).
+
+
+### North America
+
+The ressource package for Oceania at the moment includes grids from the
```

Comment:
Wouldn't "includes grids *for* the following countries" be better?

Reply:
Yes. I am changing it to "The regional package for <REGION> include resources for the following countries:"